### ISSUE TITLE
Remember the last folder feature is a bit buggy

### ISSUE THREAD

**Issue by zeromido** [drizt/torrent-file-editor#27]
My English is kinda broke here :p so if you don't understand anything please tell me.
Remember the last folder feature is a bit buggy. I can't explain it to you, but here's the detailed steps to find the bugs I'm talking about:
Click "Open" and select a file.
Navigate to "Files" tab and click "Add folder" and select a folder.
Click "Open" again. As you can see, the last folder has changed to the folder you added to the files list.
Click "Add folder" again. As you can see, it's the root (parent) of the folder you selected not the folder itself.
Note: If you clicked "Open" again the path in "Add folder" will remain as it is.
In other words: "Add folder" last folder changes "Open" last folder and "Open" doesn't change "Add folder" last folder. Tricky, huh? LOL
Also, remembering the last folder you dragged-and-dropped a file from it won't be so bad either.

**Comment by drizt** [comment]
I understand. Np.
